cli/Cargo.toml

```diff
@@ -30,7 +30,7 @@ path = "src/main.rs"
 
 [dependencies]
 clap = "2"
-cylinder = "0.2"
+cylinder = "^0.2.2"
```

Comment:
Specifying the ^ here is unnecessary as that is implied in "0.2.2".

Reply:
Fixed